### ISSUE TITLE
require('wrtc').RTCPeerConnection is undefined

### ISSUE THREAD

**Issue by piranna** [node-webrtc/node-webrtc#77]
As suggested at #76 I went to my code to do `var RTCPeerConnection = require('wrtc').RTCPeerConnection;`, just to my surprise that it was already done in my code, so it was not necesary :-) But, executing my tests on Node.js, I've got the fact that `RTCPeerConnection` is undefined:

```
[piranna@Latitude:~/Dropbox/Proyectos/qunit-cli]
 > bin/qunit-cli ../WebP2P.io/test/2peers.js 
undefined

2 peers
  ✔ Connect two peers to PubNub at the same time

/home/piranna/Dropbox/Proyectos/WebP2P.io/lib/WebP2P.js:78
    var pc = new RTCPeerConnection
             ^
TypeError: undefined is not a function
    at createPeerConnection (/home/piranna/Dropbox/Proyectos/WebP2P.io/lib/WebP2P.js:78:14)
    at HandshakeManager.<anonymous> (/home/piranna/Dropbox/Proyectos/WebP2P.io/lib/WebP2P.js:466:16)
    at HandshakeManager.EventEmitter.emit (events.js:98:17)
    at Connector_PubNub.connector._messageUnpacked (/home/piranna/Dropbox/Proyectos/WebP2P.io/lib/managers/HandshakeManager.js:62:16)
    at Connector_PubNub.<anonymous> (/home/piranna/Dropbox/Proyectos/WebP2P.io/lib/managers/Manager.js:72:21)
    at Connector_PubNub.EventEmitter.emit (events.js:95:17)
    at Array.Connector._message [as 0] (/home/piranna/Dropbox/Proyectos/WebP2P.io/lib/connectors/core/Connector.js:21:10)
    at Array.<anonymous> (/home/piranna/Dropbox/Proyectos/WebP2P.io/node_modules/pubnub/pubnub.js:707:36)
    at each (/home/piranna/Dropbox/Proyectos/WebP2P.io/node_modules/pubnub/pubnub.js:140:15)
    at _connect.xdr.success (/home/piranna/Dropbox/Proyectos/WebP2P.io/node_modules/pubnub/pubnub.js:705:25)
```

As you can see at the top of the trace, that undefined is from executing `console.log(RTCPeerConnection);` statement, showing that, efectively, it's undefined :-(

Executing `console.log(require('wrtc'));` I got

```
{ PeerConnection: [Function: PeerConnection],
  DataChannel: [Function: DataChannel],
  MediaStream: [Function: MediaStream],
  MediaStreamTrack: [Function: MediaStreamTrack] }
```

Seems that we are using the old API here... :-)

**Comment by piranna** [comment]
According to the spec, the objects that should only be exported are RTCIceCandidate, RTCPeerConnection and RTCSessionDescription.

**Comment by modeswitch** [comment]
Good catch. I can fix it when I get home, or if you want to make a pull request I'll merge it. I think the tests and examples will need updating as a result.

**Comment by piranna** [comment]
I was going to do it myself, but I don't understand the code, there are some things that I don't know if they are bugs or done in purposse. For example, on /lib/datachannel.js, it's defined and exported a DataChannel class that receive as parameter a "internalDC" (?) and later it's defined and also exported a RTCDataChannel class, overriding the first one and making useless. Seems to me that the first one create the internal representation and the second one the public API, since all their calls are redirected from RTCDataChannel class to DataChannel class, but this also makes me think why don't use only just one class and make code simpler... :-/

I'll continue reviewing it.

**Comment by piranna** [comment]
Ok, independently that the (internal) classes names should be changed (or, if possible, also merged with the public ones), I got where it was the bug :-) The package.json file main entry was pointing to `./build/Release/webrtc.node`, exposing this way the plain internal objects from the bindings instead of the ones processed by Javascript, and this was not detected by the tests because they access directly to the Javascript files (this is what gave me this inspiration to look for it). After changing the field for `lib/index.js` now it find correctly the RTCPeerConnection class.

**Comment by piranna** [comment]
Fixed by issue #78
